src/js/player.js

```diff
@@ -829,6 +834,8 @@ class Player extends Component {
       'techId': `${this.id()}_${techName}_api`,
       'videoTracks': this.videoTracks_,
       'textTracks': this.textTracks_,
+      'remoteTextTracks': this.remoteTextTracks_,
+      'remoteTextTrackEls_': this.remoteTextTrackEls_,
```

Comment:
Unfortunately, this change does need to be in this PR, some of the tests rely on tracks and will fail without it because techs are now async.